clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTaskTest.java

```diff
@@ -60,6 +87,101 @@
 
 
 public class ActiveActiveStoreIngestionTaskTest {
+  String STORE_NAME = "Thvorusleikir_store";
+  String PUSH_JOB_ID = "yule";
+  String BOOTSTRAP_SERVER = "Stekkjastaur";
+  String TEST_CLUSTER_NAME = "venice-GRYLA";
+
+  @Test
+  public void testisReadyToServeAnnouncedWithRTLag() {
+    // Set up PubSubTopicRepository
+    PubSubTopicRepository pubSubTopicRepository = mock(PubSubTopicRepository.class);
+    PubSubTopic pubSubTopic = new TestPubSubTopic(STORE_NAME + "_v1", STORE_NAME, PubSubTopicType.VERSION_TOPIC);
+    when(pubSubTopicRepository.getTopic("Thvorusleikir_store_v1")).thenReturn(pubSubTopic);
+
+    // Setup store/schema/storage repository
+    ReadOnlyStoreRepository readOnlyStoreRepository = mock(ReadOnlyStoreRepository.class);
+    ReadOnlySchemaRepository readOnlySchemaRepository = mock(ReadOnlySchemaRepository.class);
+    StorageEngineRepository storageEngineRepository = mock(StorageEngineRepository.class);
+    when(storageEngineRepository.getLocalStorageEngine(any())).thenReturn(new BlackHoleStorageEngine(STORE_NAME));
+
+    // Setup server config
+    VeniceServerConfig serverConfig = mock(VeniceServerConfig.class);
+    when(serverConfig.freezeIngestionIfReadyToServeOrLocalDataExists()).thenReturn(false);
+    when(serverConfig.getKafkaClusterUrlResolver()).thenReturn(null);
+    when(serverConfig.getKafkaClusterUrlToIdMap()).thenReturn(new Object2IntArrayMap<>());
+    when(serverConfig.getKafkaClusterIdToUrlMap()).thenReturn(new Int2ObjectArrayMap<>());
+    when(serverConfig.getConsumerPoolSizePerKafkaCluster()).thenReturn(1);
+
+    // Set up IngestionTask Builder
+    StoreIngestionTaskFactory.Builder builder = new StoreIngestionTaskFactory.Builder();
+    builder.setPubSubTopicRepository(pubSubTopicRepository);
+    builder.setHostLevelIngestionStats(mock(AggHostLevelIngestionStats.class));
+    builder.setAggKafkaConsumerService(mock(AggKafkaConsumerService.class));
+    builder.setMetadataRepository(readOnlyStoreRepository);
+    builder.setServerConfig(serverConfig);
+    builder.setSchemaRepository(readOnlySchemaRepository);
+    builder.setStorageEngineRepository(storageEngineRepository);
+
+    // Set up version config and store config
+    HybridStoreConfig hybridStoreConfig = new HybridStoreConfigImpl(
+        100L,
+        100L,
+        100L,
+        DataReplicationPolicy.ACTIVE_ACTIVE,
+        BufferReplayPolicy.REWIND_FROM_EOP);
+    Version mockVersion = new VersionImpl(STORE_NAME, 1, PUSH_JOB_ID);
+    mockVersion.setHybridStoreConfig(hybridStoreConfig);
+
+    Store store = new ZKStore(
+        STORE_NAME,
+        "Felix",
```

Comment:
Not sure how I feel about being a black hole in your mind 🤣 ...

Reply:
You're listed here as the owner, so technically you're king of the black hold!